spec/models/tag_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Tag do
   subject(:tag) { described_class.new(name: 'test') }
 
+  it { should validate_uniqueness_of(:name).case_insensitive }
```

Comment:
Is it possible to write this test without using the shoulda matchers?

Reply:
It is. I really like this gem to write unit test for model validation and other model stuff like that, but if we don't plan to test this kind of stuff I can certainly remove shoulda-matchers and rewrite the test manually.

Reply:
I guess I wonder if this is testing our code or testing Rails itself.

Reply:
Fixed, if we want to test more models validation and relation maybe think at this gem, otherwise it sounds good I think.

Reply:
Maybe we're not enough working on this codebase for this to make sense, but when you have a lot of people who works on a codebase, this is nice to avoid this kind of change to pass the code review by error. :)